app/scripts/abotypen/overview/abotypenoverview.controller.js

```diff
@@ -104,6 +127,24 @@ angular.module('openolitor-admin')
         isDisabled: function() {
           return !$scope.checkboxes.checkedAny;
         }
+      }, {
+        label: gettext('E-Mail Formular'),
+        noEntityText: true,
+        iconClass: 'glyphicon glyphicon-envelope',
+        onExecute: function() {
+          $scope.url = 'mailing/sendEmailToAbotypSubscribers';
+          $scope.message = 'Labels allowed to be replaced: \n {{person.anrede}} \n {{person.vorname}} \n {{person.name}} \n {{person.rolle}} \n {{person.kundeId}} \n {{abotyp.name}} \n {{abotyp.beschreibung}}  \n {{abotyp.preis}}  \n {{abotyp.preiseinheit}}  \n {{abotyp.laufzeiteinheit}}';  
```

Comment:
Translation:
- Text must be in German
- _gettext_ must be injected and be called

Reply:
done